_i18n/nb-no/resources/moneropedia/account.md

```diff
@@ -0,0 +1,79 @@
+---
+terms: ["konto", "kontoer", "lommebok", "lommebøker"]
+summary: "Tilsvarende en bankkonto og inneholder alle dine sendte og mottatte transaksjoner"
+---
+
```

Comment:
Please add the snippet for the versioning system: `{% include disclaimer.html translated="yes" translationOutdated="no" %}`. It has to be added to **all** moneropedia entries. See the German translation: https://github.com/monero-project/monero-site/pull/1224/files

Reply:
Done. Thanks for letting me know. I have updated my fork a couple of days ago, also, as I noticed my files where from my fork from July. I just need to update this pull (if that is possible)

Reply:
It's not really a great practice, but you can close this and make a new one if it's better for you. I think there is a way to update a fork using github's user interface, but i'm not sure about how it's done. As i was saying in the chat, using git locally is more powerful.